README.md

```diff
@@ -59,6 +66,61 @@ else {
+  + Example: [https://github.com/dwyl/style-guide/blob/<b>dea0009638b7923521a13190f17090af37a7ff22</b>/README.md](https://github.com/dwyl/style-guide/blob/dea0009638b7923521a13190f17090af37a7ff22/README.md) and _not_ https://github.com/dwyl/style-guide/blob/master/README.md
+  + To get this URL, go to the _History_ tab on the top right of your document and choosing a commit form the list that appears    
+  <img width="288" alt="history-tab-on-git-documents" src="https://cloud.githubusercontent.com/assets/4185328/9290455/55d5dc6e-438c-11e5-851d-71127213f565.png">
++ **When referring to a specific piece of code, include the line number** that code is on
+  + You can either add this by add `#L` and the line number to the end of the URL (e.g. https://github.com/dwyl/hapi-socketio-redis-chat-example/blame/b26354e3f37b2bdd0414b9b01bfe45db7ee9504e/lib/chat.js#L6) **or**
+  + Go to a specific commit (as above), click on 'View' and then click on 'Blame' in the top right hand corner    
+  <img width="274" alt="blame-tab-on-git-documents" src="https://cloud.githubusercontent.com/assets/4185328/9290470/e46c2578-438c-11e5-95a7-19dfbcf82b00.png">
+  + You can now click on any line and the line number will be added to the URL
+
+###Commits
++ Use the **third person present tense** in your commit messages, as if you were finishing off the sentence _"This commit message..."_
+  + For example _"adds riot.js to index"_ or _"fixes #12 disappearing content bug"_
++ **Include an issue number in _every commit message_**
+  + The commit message will then appear within that issue and ensure traceability of every contribution    
+  
+<img width="704" alt="commit-message-referenced-from-issue" src="https://cloud.githubusercontent.com/assets/4185328/9212670/dda6840c-4082-11e5-8e58-c4077f5ab089.png">
++ Keep your commits **'common sensibly small'**
+  + A good rule of thumb is that _if you have to use the word 'and' in your commit message, you're probably doing too much in a single commit_ unless the things you're committing are intrinsically tied together.
++ **If you're pairing**, consider giving your pair some credit in your commit messages by including their initials:
+
+<img width="723" alt="paired-commit-message" src="https://cloud.githubusercontent.com/assets/4185328/9212496/3adc19c2-4081-11e5-956c-8655c1f37946.png">
++ Did you know you can [use emojis in your commit messages](https://github.com/dwyl/start-here/issues/49)? It's totally a thing.
+<img width="695" alt="emojis-in-commit-messages" src="https://cloud.githubusercontent.com/assets/4185328/9212573/14112b74-4082-11e5-822e-bc66250c5712.png">
+
+
+
+###Pull Requests
+Good pull requests (PR) reduce the back and forth required between the person submitting the PR and the assigned reviewer, saving everyone time.
++ Before submitting a PR, **open a new issue** so you can _confirm the need_ for the functionality you are intending to add (like in [this example](https://github.com/dshaw/env/issues/6))
```

Comment:
This is just an example of opening an issue before the PR, I've linked to the ps-tree PR as a good example of a pull request itself.